File: app/src/main/java/com/jedev/leflix/service/API/Entities/VolumeInfo.java
package com.jedev.leflix.service.API.Entities;

import java.util.List;

public class VolumeInfo {
    public String title;
    public String subtitle;
    public List<String> authors;
    public String publisher;
    public String publishedDate;
    public String description;
    public List<IndustryIdentifier> industryIdentifiers;
    public ReadingModes readingModes;
    public Integer pageCount;
    public String printType;
    public List<String> categories;
    public Double averageRating;
    public Integer ratingsCount;
    public String maturityRating;
    public Boolean allowAnonLogging;
    public String contentVersion;
    public PanelizationSummary panelizationSummary;
    public ImageLinks imageLinks;
    public String language;
    public String previewLink;
    public String infoLink;
    public String canonicalVolumeLink;
    public SeriesInfo seriesInfo;
}

File: app/src/main/java/com/jedev/leflix/service/api/entities/VolumeInfo.kt
package com.jedev.leflix.service.api.entities

interface VolumeInfo {
    var title: String
    var subtitle: String
    var authors: List<String>
    var publisher: String
    var publishedDate: String
    var description: String
    var industryIdentifiers: List<IndustryIdentifier>
    var readingModes: ReadingModes
    var pageCount: Int
    var printType: String
    var categories: List<String>
    var averageRating: Double
    var ratingsCount: Int
    var maturityRating: String
    var allowAnonLogging: Boolean
    var contentVersion: String
    var panelizationSummary: PanelizationSummary
    var imageLinks: ImageLinks
    var language: String
    var previewLink: String
    var infoLink: String
    var canonicalVolumeLink: String
    var seriesInfo: SeriesInfo
}